### PR TITLE
Feature/update form autocomplete

### PR DESCRIPTION
**Pull request by YanLabCHU**

### CODE REVIEW

File: client/src/components/screens/PatientSubmission/index.jsx [hunk 1]
```diff
@@ -289,17 +289,15 @@ const Approval = ({
...
-              classeName="searchInput"
-              placeholder="Recherche par nom ou licence…"
-              dataSource={practitionerOptionsLabels}
-              onSelect={practitionerOptionSelected}
-              onChange={practitionerSearchTermChanged}
-            />,
-          )}
+          <AutoComplete
+            optionLabelProp="text"
+            classeName="searchInput"
```
Comment (plaplante-chust) [inline client/src/components/screens/PatientSubmission/index.jsx:294]: Nice franglais here :)  but I know it's not from you.

File: client/src/components/screens/PatientSubmission/index.jsx [hunk 2]
```diff
@@ -289,17 +289,15 @@ const Approval = ({
...
+            placeholder="Recherche par nom ou licence…"
```
Comment (plaplante-chust) [inline client/src/components/screens/PatientSubmission/index.jsx:295]: Will the translations be handled in another PR?

File: client/src/components/screens/PatientSubmission/index.jsx [hunk 3]
```diff
@@ -741,12 +738,12 @@ class PatientSubmissionScreen extends React.Component {
...
 
-  handlePractitionerOptionSelected(value) {
+  handlePractitionerOptionSelected(license) {
     const { actions } = this.props;
     const { practitionerOptions } = this.state;
-    const option = practitionerOptions.find(o => stringifyPractionerOption(o) === value);
+    const option = practitionerOptions.find(o => o.license === license);
 
-    if (option) {
+    if (option != null) {
```
Comment (plaplante-chust) [inline client/src/components/screens/PatientSubmission/index.jsx:746]: Here we explicitely want to check for not null?  No need to check for empty, undefined, NaN, false or 0?

**Review by plaplante-chust — COMMENTED** [review]
Just a few minor things to take a look at... let me know what you think.

File: client/src/components/screens/PatientSubmission/index.jsx (hunk 2 above)
Comment (YanLabCHU) [reply client/src/components/screens/PatientSubmission/index.jsx:295]: Exactly

File: client/src/components/screens/PatientSubmission/index.jsx (hunk 3 above)
Comment (YanLabCHU) [reply client/src/components/screens/PatientSubmission/index.jsx:746]: `option` is the result of the `find` function, which returns `undefined` if the value is not found. (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find) So there is no need to check for NaN, flase, 0 etc.
I use the `!= null` operator (notice the double equality), which include `null` and `undefined` ...

**Review by plaplante-chust — APPROVED** [review]

**Review by claudia1296 — APPROVED** [review]
LGTM
